Dockerfile.win64

```diff
@@ -18,3 +18,9 @@ RUN scons -c platform=windows
 
 RUN scons platform=windows prefix="${FINAL_TARGET_DIR}"
 
+
+WORKDIR /opt/godot-videodecoder/ffmpeg-static
+RUN ./build.sh -B -p windows -T "$THIRDPARTY_DIR/win32"
+WORKDIR /opt/godot-videodecoder
+RUN scons -c platform=win32
+RUN scons platform=win32 prefix="${FINAL_TARGET_DIR}"
```

Comment:
This looks like it should have been removed?

Reply:
Yeah, i forgot to remove it :/ I'm fixing this right now.

Reply:
Ok. Should be ok. If you don't see the changes see master branch of my fork. I don't know that i fixed it that way, you can see it in this PR. I'm very new to Github's PRs